src/test_merge_sort.py

```diff
@@ -0,0 +1,15 @@
+# coding=utf-8
+
```

Comment:
You need a few more tests than just these two. Empty lists? Repeated numbers? Consider some more edge cases.